taxi/views.py

```diff
@@ -1,3 +1,12 @@
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
+from taxi.models import Driver, Manufacturer, Car
```

Comment:
fix the imports. python modules first, then installed modules, then your own and each group should be separated by blank line